news/2023/2023-07-28-flag-change-updates.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Country Flag Change Updates"
+date: 2023-07-28 13:00:00 +0000
+---
+
+In need of a flag change? Look no further.
+
+![](/wiki/shared/news/2023-07-28-flag-change-updates/banner.jpg)
+
+For every player, there is a flag displayed on their profile reflecting their country of residence. This flag is automatically assigned during account creation, is used for country leaderboards and affects eligibility for some country-based tournaments, contests and other events.
+
+Some players may have found themselves in an awkward situation when trying to play for their country while having the wrong flag, perhaps due to moving or having created their account during a trip. In the past, this was rectifiable by contacting the [account support team](/wiki/People/Account_support_team) with proof of residency, but that system was discontinued since December 2021 due to cases of abuse.
+
+However, this has now been changed — if you've been consistently connecting from a country that differs from the country associated with your account over a long period of time (currently this is 6 months), you will be prompted to change your country flag in the [account settings page](https://osu.ppy.sh/home/account/edit):
```

Comment:
I know @peppy was adamant about not documenting the criteria but can we at least provide this much so as to not leave people guessing?